Spine/Operation.swift

```diff
@@ -211,12 +211,14 @@ class SaveOperation: Operation {
 		Spine.logInfo(.Spine, "Saving resource \(resource) using URL: \(request.URL)")
 		
 		HTTPClient.request(request.method, URL: request.URL, payload: request.payload) { statusCode, responseData, networkError in
-			if let networkError = networkError {
-				self.result = Failable(networkError)
-				self.state = .Finished
-				return
-			}
-			
+
+            if 400 ... 599 ~= statusCode! {
+                let error = NSError(domain: "networkError", code: statusCode!, userInfo: nil)
+                self.result = Failable(error)
+                self.state = .Finished
+                return
+            }
```

Comment:
Any idea why this is badly indented? In my xcode, it's perfect. Take a look.

<img width="778" alt="screen shot 2015-07-18 at 4 02 25 pm" src="https://cloud.githubusercontent.com/assets/70312/8763175/38bc03c8-2d67-11e5-8e6c-db8727a21845.png">